Makefile

```diff
@@ -4,9 +4,3 @@ test-kafka-broker-upstream-nightly:
 
 test-kafka-broker-midstream-nightly:
 	./bin/test-kafka-broker-midstream-nightly.sh || exit 1
-
-test-kafka-broker-namespaced-upstream-nightly:
-	./bin/test-kafka-broker-namespaced-upstream-nightly.sh || exit 1
-
-test-kafka-broker-namespaced-midstream-nightly:
-	./bin/test-kafka-broker-namespaced-midstream-nightly.sh || exit 1
```

Comment:
@pierDipi can we remove these `test-kafka-broker-namespaced-xyz-nightly.sh` files since we are no longer calling them here?

Reply:
Thanks!

Done